hsmodels/schemas/aggregations.py

```diff
@@ -296,3 +303,109 @@ class Config:
+        allow_mutation=False,
+    )
+
+    version: str = Field(
+        default=None, title="Version", description="The software version or build number of the model", max_length=255
+    )
+
+    name: str = Field(
+        default="Unknown Model Program",
+        max_length=255,
+        title="Model Program Name",
+        description="A string expressing the name of the model program",
+    )
+
+    programming_languages: List[str] = Field(
+        default=[],
+        max_length=100,
+        title="Programming Languages",
+        description="The programming languages that the model is written in",
+    )
+
+    operating_systems: List[str] = Field(
+        default=[],
+        max_length=100,
+        title="Operating Systems",
+        description="Compatible operating systems to setup and run the model",
+    )
+
+    release_date: date = Field(
+        default=None, title="Release Date", description="The date that this version of the model was released"
+    )
+
+    website: AnyUrl = Field(
+        default=None,
+        title='Website',
+        description='A URL to a website describing the model that is maintained by the model developers',
+    )
+
+    code_repository: AnyUrl = Field(
+        default=None,
+        title='Software Repository',
+        description='A URL to the source code repository for the model code (e.g., git, mercurial, svn, etc.)',
+    )
+
+    file_types: List[ModelProgramFile] = Field(
+        default=None, title='File Types', description='File types used by the model program'
+    )
+
+    program_schema_json: AnyUrl = Field(
+        default=None,
+        title='Model program schema',
+        description='A url to the JSON metadata schema for the model program',
+    )
+
+    _parse_file_types = root_validator(pre=True, allow_reuse=True)(parse_file_types)
+
+
+class ModelInstanceMetadata(BaseAggregationMetadata):
+    """
+    A class used to represent the metadata associated with a model instance aggregation
+    """
+
+    class Config:
+        title = 'Model Instance Aggregation Metadata'
+
+        schema_config = {'read_only': ['type', 'url'], 'dictionary_field': ['additional_metadata']}
+
+    type: AggregationType = Field(
+        const=True,
+        default=AggregationType.ModelInstanceAggregation,
+        title="Aggregation type",
+        description="A string expressing the aggregation type from the list of HydroShare aggregation types",
+        allow_mutation=False,
+    )
+
+    includes_model_output: bool = Field(
+        title="Includes Model Output",
+        description="Indicates whether model output files are included in the aggregation",
+    )
+
+    executed_by: AnyUrl = Field(default=None, title="Executed By", description="A url to the Model Program")
+
+    program_schema_json: AnyUrl = Field(
+        default=None, title="Schema JSON url", description="A url to the Model Program schema json"
```

Comment:
I suggest:
title = "JSON Metadata schema URL"
description = "A URL to the JSON metadata schema for the related model program"